docs/api/devguide/activitylogguide.md

```diff
@@ -0,0 +1,227 @@
+      }
+    }
+  ]
+}
+```
+
+The storage mode decides how the features will be stored.
+
+* FEATURE_ONLY: Will store features with some history relative properties (defined below).
+* DIFF_ONLY: Will store features with a 'diff'.'ops' property in the XYZ namespace, containing the RFC-6902 diff to its previous object. The features after the HEAD will only contain the XYZ namespace properties.
+* FULL: Will store features with some history relative properties and a 'diff'.'ops' property in the XYZ namespace, containing the RFC-6902 diff to its previous object.
+
+**IMPORTANT**: Since it is not possible to write changes for *<span style="color:red">DeleteFeaturesByTagEvent</span>*, this event type is completely forbidden and the support of this event in the current space is removed.
+
+Within the ModifySpaceEvent, the listener prepares everything and modifies the space to log changes. It will create a new space where the features will be written to, register a new listener that will actually write the modifications and add a processor to prohibit the DeleteByTagEvents.  
+Since the ModifyFeatureEvent.response is picked up in a listener, the actual features to log are written asynchronously to the newly created space. As a result, the performance of the actual request is not decreased.
+
+A full space definition with this feature enabled looks like this:
+
+``` javascript
+{
+  "id": "mySpaceId",
+  "title": "Test space to track changes.",
+  "description": null,
+  "owner": "HERE-12345678-1234-1234-1234-f1c02cc19c37",
+  "enableUUID": true,
+  "listeners": [
+    {
+      "id": "activity-log",
+      "params": {
+        "states": 5,
+        "storageMode": "DIFF_ONLY"
+      },
+      "eventTypes": [
+        "ModifySpaceEvent.request"
+      ]
+    },
+    {
+      "id": "activity-log-writer",
+      "params": {
+        "spaceId": "someAuditSpaceId",
+        "states": 5,
+        "storageMode": "DIFF_ONLY"
+      },
+      "eventTypes": [
+        "ModifyFeaturesEvent.response"
+      ]
+    },
+    ...
+  ],
+  "processors": [
+    {
+      "id": "activity-log-writer",
+      "eventTypes": [
+        "DeleteFeaturesByTagEvent.request"
+      ]
+    },
+    ....
+  ]
+}
+```
+
+## Written features
+***
+The features written to the new space are stored by uuid. So the space with this feature enabled needs to have **enableUUID** set to true.
+
+Additionally, the original feature element will be slightly modified:
+
+* The id of the feature, will be the **uuid** of the modified feature in the original space.
+* The original properties of **'@ns:com:here:xyz'** will be stored under '@ns:com:here:xyz'.**original**
+* The original id of the feature will be stored under '@ns:com:here:xyz'.**original**.**id** within the properties.
+* A new property will be added under '@ns:com:here:xyz'.**original**.**invalidAt** to indicate the time range, when this object was the HEAD (newest) history object.
```

Comment:
Query - why "invalidAt"? The name doesn't seem to match the intent. Also, what kind of data is this? You say "time range", is it in an array of timestamps?

Reply:
It's not the latest version of the documentation. E.g., the latest wording is invalidatedAt, not invalidAt.